composer.json

```diff
@@ -13,13 +13,13 @@
     ],
     "require": {
         "cakephp/orm": "~3.0",
-        "php": ">=5.5"
+        "php": ">=5.5",
+        "php-coveralls/php-coveralls": "^2.0"
```

Comment:
Increasing the coveralls version moves the binary to `vendor/bin/php-coveralls` I think.